cli/crates/cli/Cargo.toml

```diff
@@ -62,7 +62,12 @@ tempfile = "3"
 tokio = { version = "1", features = ["full"] }
 regex = "1"
 rstest = "0.17"
+rusoto_core = "0.48"
+rusoto_dynamodb = "0.48"
```

Comment:
Could these be optional depending on the feature flag?

Reply:
Would be nice but:
```
  dev-dependencies are not allowed to be optional: `rusoto_core`
````

Reply:
Ah, I didn't spot these were under dev-deps (we have _a lot_ of dev deps) - never mind